bindings/c/test/mako/mako.cpp

```diff
@@ -181,11 +183,11 @@ int populate(Database db,
 	auto watch_trace = Stopwatch(watch_total.getStart());
 
 	Transaction systemTx = db.createTransaction();
-	for (int i = 0; i < args.tenants; ++i) {
+	for (int i = 0; i < args.total_tenants; ++i) {
 		while (true) {
 			// Until this issue https://github.com/apple/foundationdb/issues/7260 is resolved
 			// we have to commit each tenant creation transaction one-by-one
-			// while (i % 10 == 9 || i == args.tenants - 1) {
+			// while (i % 10 == 9 || i == args.total_tenants - 1) {
 			std::string tenant_name = "tenant" + std::to_string(i);
 			Tenant::createTenant(systemTx, toBytesRef(tenant_name));
```

Comment:
[The PR](https://github.com/apple/foundationdb/pull/7321) to create multiple tenants in the same transaction has merged, so we can update this logic to create tenants in batches.